workflow_scripts/protobuf/build_protobuf_unix.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: Apache-2.0
+
+export CORE_NUMBER=$1
+export INSTALL_PROTOBUF_PATH=$2
+
+if [[ -z "$CORE_NUMBER" ]]; then
+   export CORE_NUMBER=1
+fi
+
+if [[ -z "$INSTALL_PROTOBUF_PATH" ]]; then
+   export INSTALL_PROTOBUF_PATH=/usr
+fi
+
+# Build protobuf from source with -fPIC on Unix-like system
+ORIGINAL_PATH=$(pwd)
+cd ..
+git clone https://github.com/protocolbuffers/protobuf.git
```

Comment:
Fetch the tarball would be simpler, which requires less network traffic than "git clone". When you run "git clone", you download all the versions.